GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -452,6 +478,22 @@ internal override void EIEvtcParse(ulong gw2Build, int evtcVersion, FightData fi
             {
                 agentData.Refresh();
             }
+            // Gravity Ball - Timecaster gadget
+            var gravityBalls = combatData.Where(x => x.DstAgent == 0 && x.IsStateChange == ArcDPSEnums.StateChange.MaxHealthUpdate).Select(x => agentData.GetAgent(x.SrcAgent, x.Time)).Where(x => x.Type == AgentItem.AgentType.Gadget && x.HitboxHeight == 300 && x.HitboxWidth == 100 && x.Master == null).ToList();
```

Comment:
This block should be before agentData.Refresh()

Reply:
I was moving things around and didn't notice i moved it under the refresh :x